lib/krane/cli/deploy_command.rb

```diff
@@ -53,7 +55,7 @@ def self.from_options(namespace, context, options)
         if options['protected-namespaces'].size == 1 && %w('' "").include?(options['protected-namespaces'][0])
           protected_namespaces = []
         end
-
+        options[:filenames] << "-" if options['std-in']
```

Comment:
If/until we manage to fix the underlying issue with Thor forcing us to do this, this might be worth an explanatory comment. At first blush, this won't make sense to people unfamiliar with why we were compelled to do this.